upstairs/src/volume.rs

```diff
@@ -481,6 +481,12 @@ impl Volume {
             return Ok(());
         }
 
+        let bs = self.get_block_size().await?;
```

Comment:
Do we want the same check for the read path as well?

Reply:
Yes! see 156eb29